### PR TITLE
[Checkbox] feat: add checkbox disabled

### PR DESCRIPTION
**Pull request by warmwhiten**
## Description

I created a new feature to disable the checkbox and basic contents of readme.md

## Related Issues

X

## Tests

I added the following tests:

_Replace this with a list of the tests that you added as part of this PR. A change in behavior with no test covering it
will likely get reverted accidentally sooner or later. PRs must include tests for all changed/updated/fixed behaviors. See [Test Coverage](https://codecov.io/gh/dooboolab/dooboo-ui)._

## Checklist

Before you create this PR confirms that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide](https://github.com/dooboolab/dooboo-ui/blob/master/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [x] Run `yarn test` or `yarn test -u` if you need to update snapshot.
- [x] Run `yarn lint`
- [x] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

File: main/Checkbox/index.tsx [hunk 1]
```diff
@@ -84,15 +91,14 @@ const CheckboxGroup: FC<CheckboxGroupProps> = ({
   };
 
   return (
-    <Container>
+    <Container style={{flexWrap:"wrap"}}>
```
Comment (p-iknow) [inline main/Checkbox/index.tsx]: For flexWrap styles, it would be more consistent to add to the StyledComponent side at the top.
(flexWrap 스타일의 경우 위쪽에 StyledComponent 쪽에 추가하는게 더 일관성이 있을 거 같아요.)

```js
const Container = styled.View`
 flex-direction: row;
 align-items: center;
 // like this
 flex-wrap: wrap
`;
```

**Review by p-iknow — COMMENTED** [review]
Thank you for your hard work. ^^

File: main/Checkbox/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# Check Box
+
+> Simple check box for react-native.
+> Refer : <https://material-ui.com/components/checkboxes>
+
```
Comment (samslow) [inline main/Checkbox/README.md]: I recommend you to make a separate PR for docs work that is not related to this PR.

File: main/Checkbox/index.tsx (hunk 1 above)
Comment (warmwhiten) [reply main/Checkbox/index.tsx]: Thanks so much for your good review! I reflected the feedback you gave me.

리뷰 감사합니다. 피드백 주신 것 반영했어요!

File: stories/dooboo-ui/Checkbox.stories.tsx [hunk 3]
```diff
@@ -57,6 +57,40 @@ function Default(): React.ReactElement {
...
+              selected: true,
+              disabled: true,
+            },
+            {
+              label: 'unchecked disabled value',
+              value: 4,
+              selected: false,
+              disabled: true,
+            },
+          ]}
```
Comment (samslow) [inline stories/dooboo-ui/Checkbox.stories.tsx]: It's not a necessary modification, but how about putting the data-related part like this at the top?

File: main/Checkbox/index.tsx [hunk 4]
```diff
@@ -30,6 +30,7 @@ const COLOR: {
 const Container = styled.View`
  flex-direction: row;
  align-items: center;
+ flew-wrap: wrap;
```
Comment (p-iknow) [inline main/Checkbox/index.tsx]: "flew-wrap" should be edited to "flex-wrap"

**Review by p-iknow — CHANGES_REQUESTED** [review]

File: main/Checkbox/index.tsx [hunk 5]
```diff
@@ -138,8 +138,7 @@ const Checkbox: FC<CheckboxProps> = ({
             />}
           </Marker>
         </MarkerContainer>
-        <Label labelSize={labelSize} labelColor={isDisabled ? COLOR.LIGHTGRAY : labelColor}>
```
Comment (JinleeJeong) [inline main/Checkbox/index.tsx]: Is there a reason for this subspace? I think the existing is better for readability!

File: main/Checkbox/index.tsx [hunk 6]
```diff
@@ -111,30 +117,28 @@ const Checkbox: FC<CheckboxProps> = ({
...
         >
           <Marker isSelected={isSelected}>
             {isSelected && <MarkerImg
               source={require('../__assets__/check_tick.png')}
             />}
           </Marker>
         </MarkerContainer>
-        <Label labelSize={labelSize} labelColor={labelColor}>
-          {item.label}
+        <Label labelSize={labelSize} labelColor={isDisabled ? COLOR.LIGHTGRAY : labelColor}>          {item.label}
```
Comment (JinleeJeong) [inline main/Checkbox/index.tsx]: Is there a reason for this subspace? I think the existing is better for readability

File: main/Checkbox/README.md (hunk 2 above)
Comment (warmwhiten) [reply main/Checkbox/README.md]: I didn't think about this, but I found out thanks to you ✨ I'll reflect this in the next commit

File: stories/dooboo-ui/Checkbox.stories.tsx (hunk 3 above)
Comment (warmwhiten) [reply stories/dooboo-ui/Checkbox.stories.tsx]: I think it is good for the readability of the code. thanks! 👍

File: main/Checkbox/index.tsx [hunk 7]
```diff
@@ -63,11 +72,10 @@ interface CheckboxProps {
   boxSize?: number;
   boxColor?: string;
   defaultChecked?: boolean;
-  disabled?: boolean;
   onPress?: (value: string | number) => void;
   labelSize?: number;
   labelColor?: string;
-  item: { label: string; value: string | number; selected: boolean; }
+  item: { label: string; value: string | number; selected: boolean; disabled?: boolean;}
```
Comment (JeffGuKang) [inline main/Checkbox/index.tsx]: I would be great to make the new interface for the checkbox data type. 
for example

```
interface CheckboxItem {
  label: string,
 ...
}
```

And you will replace this line as below. 

```
item: CheckboxItem;
```

And it can be used here as well.

https://github.com/dooboo-ui-turtle/dooboo-ui/blob/a11d9380cd879472d56a3176218984d1058d3d37/main/Checkbox/index.tsx#L64

File: stories/dooboo-ui/Checkbox.stories.tsx [hunk 8]
```diff
@@ -27,6 +27,49 @@ const Selected = styled.Text`
   font-weight: bold;
 `;
 
+const defaultCheckboxData: any[] = [
```
Comment (JeffGuKang) [inline stories/dooboo-ui/Checkbox.stories.tsx]: It can be `const defaultData:  CheeckboxItem[]` if you defined `CheckboxItem` interface.

File: stories/dooboo-ui/Checkbox.stories.tsx [hunk 9]
```diff
@@ -27,6 +27,49 @@ const Selected = styled.Text`
...
+    selected: false,
+  },
+  {
+    label: 'third',
+    value: 3,
+    selected: false,
+  },
+];
+
+const disabledCheckboxData: any[] = [
```
Comment (JeffGuKang) [inline stories/dooboo-ui/Checkbox.stories.tsx]: It can be `const defaultData:  CheeckboxItem[]` if you defined `CheckboxItem` interface.

File: stories/dooboo-ui/Checkbox.stories.tsx [hunk 10]
```diff
@@ -39,24 +82,19 @@ function Default(): React.ReactElement {
...
-              value: 2,
-              selected: false,
-            },
-            {
-              label: 'third',
-              value: 3,
-              selected: false,
-            },
-          ]}
+          selectedValue={defaultCheckboxData}
```
Comment (JeffGuKang) [inline stories/dooboo-ui/Checkbox.stories.tsx]: I think `items` is better props name than `selectedValue` as a single `Checkbox`'s thing.

Comment (JinleeJeong) [reply stories/dooboo-ui/Checkbox.stories.tsx]: Thank you for the detailed review. It's corrected. Any further review would be appreciated. 👍
